nbgdrive/static/tree.js

```diff
@@ -34,57 +34,102 @@ define([
-                                                             .text('User authenticated!')
-                                            )
-                                        );
-
-                                        /* Alert user that they've successfully authenticated. */
-                                        $("#nbgdrive-authenticated-result").fadeOut(4000);
-
-                                        createButtonGroup();
-
-                                        /* Add a button to allow to manually sync their Drive files. */
-                                        createManualSyncButton();
-
-                                        /* Add a button to allow pulling from gdrive. */
-                                        createManualPullButton();
-
-                                        /* Add a button to allow to logout from gdrive. */
-                                        createLogoutButton();
-
-                                        /* GET Request to alert Server to create a sync directory. */
-                                        $.getJSON(utils.get_body_data('baseUrl') + 'createDrive', function(data) {
-                                            var display = String(data['status']);
-                                        });
-
-                                    } else {
-                                        $('#notebook_toolbar').append(
-                                            $('<div>').attr('id', 'nbgdrive-display')
-                                                    .addClass('btn-group')
-                                                    .addClass('pull-right')
-                                            .append(
-                                                $('<strong>').attr('id', 'nbgdrive-authenticated-result')
-                                                             .text('Your key was incorrect. Please try again!')
-                                            )
-                                        );
-
-                                        $("#nbgdrive-authenticated-result").fadeOut(4000);
-                                        setTimeout(displayDriveVerificationLink, 5000);
+                                                .append(
+                                                    $('<input>').attr('id', 'nbgdrive-folder-path')
+                                                        .attr('type', 'text')
+                                                ).append(
+                                                    $('<button>').attr('id', 'nbgdrive-button')
+                                                        .text('Submit Folder Name')
+                                                        .click(function() {
+
+                                                            var folderPath = $("#nbgdrive-folder-path").val();
+
+                                                            console.log('okkk')
```

Comment:
Lol, should we remove this in prod? Or change text to be something like `console.log("Synching to specified path: " + folderPath)`?